packages/office-ui-fabric-react/src/components/Coachmark/PositioningContainer/PositioningContainer.tsx

```diff
@@ -1,438 +1,395 @@
 import * as React from 'react';
-import { IPositioningContainerProps } from './PositioningContainer.types';
 import { getClassNames } from './PositioningContainer.styles';
+import { ZIndexes } from '../../../Styling';
 import { Layer } from '../../../Layer';
 
 // Utilites/Helpers
 import { DirectionalHint } from '../../../common/DirectionalHint';
-import {
-  Point,
-  IRectangle,
-  css,
-  elementContains,
-  focusFirstChild,
-  getWindow,
-  getDocument,
-  initializeComponentRef,
-  Async,
-  EventGroup,
-} from '../../../Utilities';
-
-import {
-  getMaxHeight,
-  positionElement,
-  IPositionedData,
-  IPositionProps,
-  IPosition,
-  RectangleEdge,
-} from '../../../utilities/positioning';
+import { css, elementContains, focusFirstChild, EventGroup, getPropsWithDefaults } from '../../../Utilities';
+
+import { getMaxHeight, positionElement, RectangleEdge } from '../../../Positioning';
 
 import { AnimationClassNames, mergeStyles } from '../../../Styling';
+import { useMergedRefs, useAsync, useTarget } from '@fluentui/react-hooks';
+import type { IPositioningContainerProps } from './PositioningContainer.types';
```

Comment:
These `import type` changes also need to be reverted. (Same in all other files.)